src/components/StoredQueryTab/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+type DatasetStoredQueryProps = {
```

Comment:
I think this may be the cause of the Jest failures. We'll probably need to convert the file extension to .tsx before using typescript types